music_assistant/server/providers/deezer/__init__.py

```diff
@@ -147,11 +158,16 @@ async def handle_setup(self) -> None:
             access_token=self.config.get_value(CONF_ACCESS_TOKEN),  # type: ignore
         )
         try:
-            deezer_client = await DeezerClient.get_deezer_client(self=None, creds=self.creds)
-            self.client = DeezerClient(creds=self.creds, client=deezer_client)
+            self.client = deezer.Client(
```

Comment:
there is no hidden IO in the instantiating of the Client ?

Reply:
No, but that makes me wonder why i ever added the try catch thing...